src/test-console/Program.cs

```diff
@@ -63,6 +66,27 @@ private static void Main(string[] args)
             // InitializeRequest_Setup();
         }
 
+
+        private async static void CreateRecepient()
+        {
+
+
+            
+
+
+            //TransfersApi api = new TransfersApi(_api);
+            TransferRecipientsApi tra = new TransferRecipientsApi(_api);
+            var result = await Task.Run(() => tra.Create("Jethro Daniel Ekozin", "0692105364", "032"));
```

Comment:
`TransferRecipientApi` object is available from the `_api.Transfers.Recipients` property.  As such, this code could be reduced to:

```C#
var tra = _api.Transfers.Recipients.Create("Jethro Daniel Ekozin", "0692105364", "032");
```

Cleaner, right?